lib/screens/feedback_page.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:flutter/material.dart';
+import 'package:innopolis_feedback/data.dart';
+import 'package:innopolis_feedback/shared/styles.dart';
+
+class FeedbackPage extends StatefulWidget {
+  final StudentFeedback f;
+
+  FeedbackPage(this.f);
+
+  @override
+  _FeedbackPageState createState() => _FeedbackPageState();
+}
+
+class _FeedbackPageState extends State<FeedbackPage> {
+  List<FeedbackComment> comments = [];
+  String uid, email;
+
+  updateComments(List<FeedbackComment> c) {
+    setState(() {
+      comments = c;
+    });
+  }
+
+  fetchComments() {
+    getComments(widget.f.feedbackId).listen((c) {
+      updateComments(c);
+    });
+  }
+
+  @override
+  void initState() {
+    fetchComments();
+    uid = FirebaseAuth.instance.currentUser.uid;
+    email = FirebaseAuth.instance.currentUser.email;
+    super.initState();
+  }
+
+
+  String prettyDate(DateTime date) =>
+      '${date.hour}:${date.minute} ${date.day}.${date.month}.${date.year}';
+
+  Widget commentToWidget(FeedbackComment c) => ListTile(
+        title: Text(c.email),
+        subtitle: Text(c.text),
+        trailing: Text(prettyDate(c.date)),
+      );
+
+  final commentHolder = TextEditingController();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(title: Text(widget.f.email), centerTitle: true,),
+        body: Column(
+          children: [
+            Text(widget.f.message),
+            TextField(
+              controller: commentHolder,
+              onSubmitted: (String comment) {
+                submitComment(widget.f,
+                    FeedbackComment('', uid, email, DateTime.now(), comment));
+                commentHolder.clear();
+              },
+              decoration: textInputDecoration.copyWith(hintText: 'Comment'),
+            ),
```

Comment:
There should be a submit button to add the comment. This is not a chat area.

Reply:
well, I'll add it. Which widget is better to use there?

Reply:
RaisedButton would work fine, I guess.

Reply:
used another solution, u could check

Reply:
It is nice 👌 but I am not sure why the TextField has a red border when typing.